api/config/sync-catalog/32-sync-catalog-cronjob.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: catalog-refresh
+  namespace: tekton-hub
+  labels:
+    app: api
+spec:
+  schedule: "*/30 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: catalog-refresh
+              image: curlimages/curl:7.71.1
+              env:
+                - name: Token
+                  valueFrom:
+                    secretKeyRef:
+                      name: catalog-refresh
+                      key: Token
+              args: ["-i", "-X", "PUT", "-H", "Authorization:$(Token)", "api:8000/catalog/refresh"]
```

Comment:
wouldn't 8000 fail?

Reply:
Why? API server runs on 8000.

Reply:
@SM43 I thought the service is reachable on 80but it isn't...